demo/src/pages/interactive-examples.js

```diff
@@ -0,0 +1,280 @@
+              margin-right: 1rem;
+              margin-bottom: 0.5rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.NORMAL}
+            size={Button.SIZE.EXTRA_SMALL}
+          >
+            Extra Small Normal
+          </Button>
+          <br />
+          <Button
+            css={css`
+              margin-right: 1rem;
+              margin-bottom: 0.5rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.OUTLINE}
+          >
+            Outline Button
+          </Button>
+          <Button
+            css={css`
+              margin-right: 1rem;
+              margin-bottom: 0.5rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.OUTLINE}
+            size={Button.SIZE.SMALL}
+          >
+            Small Outline
+          </Button>
+          <Button
+            css={css`
+              margin-right: 1rem;
+              margin-bottom: 0.5rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.OUTLINE}
+            size={Button.SIZE.EXTRA_SMALL}
+          >
+            Extra Small Outline
+          </Button>
+          <br />
+          <Button
+            css={css`
+              margin-right: 1rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.LINK}
+          >
+            Link Button
+          </Button>
+          <Button
+            css={css`
+              margin-right: 1rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.LINK}
+            size={Button.SIZE.SMALL}
+          >
+            Small Link
+          </Button>
+          <Button
+            css={css`
+              margin-right: 1rem;
+              width: 125px;
+            `}
+            variant={Button.VARIANT.LINK}
+            size={Button.SIZE.EXTRA_SMALL}
+          >
+            Extra Small Link
+          </Button>
+        </PropsDisplay>
+        <div
+          css={css`
+            margin-bottom: 2em;
+          `}
+        />
+        <PropsDisplay componentInfo={signupModalData}>
+          <Button
+            variant={Button.VARIANT.PRIMARY}
+            onClick={() => setShowModal(true)}
+          >
+            Display Sign Up Modal
+          </Button>
+          <SignupModal isOpen={showModal} onClose={() => setShowModal(false)} />
+        </PropsDisplay>
+        <div
+          css={css`
+            margin-bottom: 2em;
+          `}
+        />
+        <PropsDisplay componentInfo={linkData}>
+          <Link to="https://newrelic.com/">Link to newrelic.com</Link>
+          <br />
+          <div
+            css={css`
+              padding-bottom: 0.5rem;
+            `}
+          />
+          <Link displayExternalIcon to="https://newrelic.com/">
+            Link to newrelic.com with External Icon
+          </Link>
+        </PropsDisplay>
+      </Layout.Content>
+    </Layout.Main>
+  );
+};
+
```

Comment:
Small nitpick, we should set PropTypes here to satisfy the linter. Thankfully, @mary-martinez already wrote a PropType utility, so this should be all you need:
```suggestion

InteractiveExamples.propTypes = {
  data: docgenJson,
};

```

Reply:
updated